fe/fe-core/src/main/java/org/apache/doris/catalog/Table.java

```diff
@@ -115,6 +117,9 @@ public abstract class Table extends MetaObject implements Writable, TableIf {
     // sql for creating this table, default is "";
     protected String ddlSql = "";
 
+    @SerializedName(value = "constraints")
```

Comment:
@SerializedName annotation use for what?

Reply:
useless